.gitignore

```diff
@@ -1,4 +1,5 @@
 public
 resources
 assets/jsconfig.json
-.hugo_build.lock
\ No newline at end of file
+.hugo_build.lock
```

Comment:
Can you please remove the changes made to `.gitignore`?

Reply:
I did it.

Reply:
Should I have left the exception for temporary folder `.idea`?

Reply:
> Should I have left the exception for temporary folder `.idea`?

No, it's fine like this. Thank you!

Reply:
You're welcome.